packages/ember-htmlbars/tests/helpers/get_test.js

```diff
@@ -1,495 +0,0 @@
-import Ember from 'ember-metal/core';
-import EmberObject from 'ember-runtime/system/object';
-import run from 'ember-metal/run_loop';
-import compile from 'ember-template-compiler/system/compile';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import EmberView from 'ember-views/views/view';
-import ComponentLookup from 'ember-views/component_lookup';
-import TextField from 'ember-views/views/text_field';
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, owner;
-
-QUnit.module('ember-htmlbars: {{get}} helper', {
-  setup() {
-    owner = buildOwner();
-    owner.register('component:-text-field', TextField);
-    owner.register('component-lookup:main', ComponentLookup);
-    owner.registerOptionsForType('template', { instantiate: false });
-  },
-  teardown() {
-    run(function() {
-      Ember.TEMPLATES = {};
-    });
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-  }
-});
-
-QUnit.test('should be able to get an object value with a static key', function() {
-  var context = {
-    colors: { apple: 'red', banana: 'yellow' }
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors \'apple\'}}] [{{if true (get colors \'apple\')}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\' for {{get colors \'apple\'}}');
-
-  run(function() {
-    view.set('context.colors', { apple: 'green', banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\' for {{get colors \'apple\'}}');
-
-  run(function() {
-    view.set('context.colors.apple', 'red');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\' for {{get colors \'apple\'}}');
-});
-
-QUnit.test('should be able to get an object value with nested static key', function() {
-  var context = {
-    colors: { apple: { gala: 'red and yellow' }, banana: 'yellow' }
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile(`[{{get colors "apple.gala"}}] [{{if true (get colors "apple.gala")}}]`)
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red and yellow] [red and yellow]', 'should return \'red and yellow\' for {{get colors "apple.gala"}}');
-
-  run(function() {
-    view.set('context.colors', { apple: { gala: 'yellow and red striped' }, banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[yellow and red striped] [yellow and red striped]', 'should return \'yellow and red striped\' for {{get colors \'apple.gala\'}}');
-
-  run(function() {
-    view.set('context.colors.apple.gala', 'yellow-redish');
-  });
-
-  equal(view.$().text(), '[yellow-redish] [yellow-redish]', 'should return \'yellow-redish\' for {{get colors \'apple.gala\'}}');
-});
-
-QUnit.test('should be able to get an object value with a bound/dynamic key', function() {
```

Comment:
:heavy_check_mark: should be able to get an object value with a bound/dynamic key